source/blog/2020-11-17-dbal-3.0.0.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "New Major Release: Doctrine DBAL 3.0"
+authorName: Benjamin Eberlei
+authorEmail: kontakt@beberlei.de
+permalink: /2020/11/17/dbal-3.0.0.html
+---
+
+We have released a new major version of Doctrine DBAL, version 3.0,0. This new
+major version comes almost 10 years after DBAL 2.0 was released on December
+2010, then coupled into the ORM 2.0.
+
+Today Doctrine DBAL is released independent of the ORM, thanks to Composer and
+sees its new major version before the ORM.
+
+This release was made possible foremost by
+[Sergei](https://twitter.com/srgmrzv), our primary DBAL maintainer, who has
```

Comment:
There's nothing wrong with just Sergei but I wouldn't mind being referred by the full name.